app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="flex flex-col items-center justify-center">
+  <h2 class="text-4xl text-center">Log in</h2>
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
```

Comment:
- [ ] While making the changes suggested by the previous reviewer, when I access the `users/sign_in` route I get the error below so make sure that you fix this :point_down:  
  
  ![image](https://github.com/Caren-Koroeny/caren_blog-app/assets/92303891/cd03b300-5126-4459-bca8-150c93be8186)

- [ ] Also,  when I access the `users/sign_up` route I get the below error so make sure that you fix this so that a new user can sign up :point_down:  

  ![image](https://github.com/Caren-Koroeny/caren_blog-app/assets/92303891/1a9a2707-a1af-4135-b816-4f00c1cd53bb)

Reply:
@Owusu-Desmond , thank you for the review. I have carefully gone through the project again, when I run it on my end this is how it looks like.
![Screenshot from 2023-06-24 21-23-44](https://github.com/Caren-Koroeny/caren_blog-app/assets/55842414/bc58a031-8c0f-4e48-bd7a-cc326c6ea3d7)

Where could the issue be, please help me.Thank you.

Reply:
@Owusu-Desmond I have tried again and i still get the same output.

![Screenshot from 2023-06-24 23-25-52](https://github.com/Caren-Koroeny/caren_blog-app/assets/55842414/4eb296a7-97b6-43ea-87fe-f45c5e479f02)

Reply:
@Owusu-Desmond on my end all is working well.Attached is the screenshots.

![Screenshot from 2023-06-24 23-30-57](https://github.com/Caren-Koroeny/caren_blog-app/assets/55842414/829d27ad-11c0-4bfc-b760-be727921c880)

![Screenshot from 2023-06-24 23-32-51](https://github.com/Caren-Koroeny/caren_blog-app/assets/55842414/2737b22b-bdb6-4426-86f1-6eacf9032c0b)